FEATURES.md

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
Let's make this "Proposing Features" or "Feature Proposals" and match in README.

Reply:
Add a TOC?